frontend-react/package.json

```diff
@@ -5,6 +5,7 @@
     "npmClient": "yarn",
     "proxy": "http://127.0.0.1:7071",
     "dependencies": {
+        "@floating-ui/react": "^0.23.0",
```

Comment:
Out of curiosity, how did you land on this library to use?

Reply:
I have previous experience with tippyjs and when looking at their docs about website support they recommended @floating-ui/react. The library used is completely up for debate, as long as it is in active development.